batter/game/output_service.py

```diff
@@ -23,7 +23,7 @@ def __init__(self, screen):
     def clear_screen(self):
         """Clears the Asciimatics buffer for the next rendering.""" 
         self._screen.clear_buffer(7, 0, 0)
-        self._draw_border()
+        # self._draw_border()
```

Comment:
make sure you uncomment this. The border is static and not an object or set of objects.